template/assessment/skeleton/src/main/resources/template.sw.yaml

```diff
@@ -0,0 +1,51 @@
+id: ${{ values.workflowId }}
+version: '${{ values.version}}'
+specVersion: '0.8'
+name: '${{ values.workflowId }}'
+description: '${{ values.description }}'
+annotations:
+  - "workflow-type/assessment"
+dataInputSchema:
+  failOnValidationErrors: true
+  schema: schemas/${{ values.artifactId }}-schema.json
+start: StartState
+extensions:
+  - extensionid: workflow-output-schema
+    outputSchema: schema/workflow-options-output-schema.json
+functions:
+  - name: execute
+    type: custom
+    operation: "service:java:com.redhat.parodos.workflow.assessment.BasicAssessment::execute"
+  - name: logOuput
+    type: custom
+    operation: "sysout:INFO"
+  - name: preCheck
+    operation: "specs/precheck.yaml#createResource_preCheck"
+states:
+  - name: StartState
+    type: operation
+    stateDataFilter:
+      output: '. +={workflowOptions: {migrationOptions: [{id: "move2kube", name: "Move2Kube workflow"}]}}'
```

Comment:
This should be the output of the `execute` function instead, correct?
If this is the case, it should be at least documented how to connect the infrastructure workflows to the assessment. Maybe we can use template variables to start a default infrastructure workflow (created from the other template) instead of hardcoding the "move2kube" one